go.mod

```diff
@@ -167,10 +168,9 @@ require (
 	github.com/mattn/go-isatty v0.0.14 // indirect
 	github.com/mattn/go-runewidth v0.0.13 // indirect
 	github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369 // indirect
-	github.com/minio/argon2 v1.0.0 // indirect
 	github.com/minio/colorjson v1.0.1 // indirect
 	github.com/minio/filepath v1.0.0 // indirect
-	github.com/minio/mc v0.0.0-20220302011226-f13defa54577 // indirect
+	github.com/minio/mc v0.0.0-20220407151251-dbc09a8bf054 // indirect
```

Comment:
@anjalshireesh I wonder if you modified this manually or this is just from go mod tidy command

Reply:
@vadmeste I had added it manually using `go get github.com/minio/mc`. To fix this, I have now sent a console PR https://github.com/minio/console/pull/1832 . After that is merged, I will change this PR to just update the console dependency